wechat-exporter.py

```diff
@@ -431,7 +431,7 @@ def save_log(self):
         fo = bz2_open(fpath + '.csv.bz2', 'wt', encoding='utf8')
       else:
         fo = open(fpath + '.csv', 'w', encoding='utf8')
-      wt = csv_writer(fo)
+      wt = csv_writer(fo, quoting=csv_QUOTE_ALL, lineterminator='\n')
```

Comment:
Per [RFC 4180](https://tools.ietf.org/html/rfc4180#section-2), lines of a CSV should be delimited by CRLF, so the default behavior of `csv.writer` is fairly good.